make/go.mk

```diff
@@ -19,4 +19,12 @@ $(OUT_DIR)/operator:
 
 .PHONY: vendor
 vendor:
-	$(Q)go mod vendor
\ No newline at end of file
+	$(Q)go mod vendor
+
+TEKTON_INSTALLATION_CR_DIR=deploy/tekton
+
+.PHONY: generate-assets
+generate-assets:
+	@echo "generating assets bindata..."
+	@go install github.com/go-bindata/go-bindata/
+	@$(GOPATH)/bin/go-bindata -pkg tekton -o pkg/resources/tekton/tekton_assets.go -nocompress -prefix $(TEKTON_INSTALLATION_CR_DIR) $(TEKTON_INSTALLATION_CR_DIR)
```

Comment:
Suggestion: I would not scatter generated assets in different directories rather I would put all generated assets in one directory.  Here tekton assert going in `pkg/resource/tekton/` while che will go in `pkg/resources/che` rather I will put all generated content in `pkg/assets/` OR `/pkg/resources/`.

Reply:
ok, let me try that

Reply:
please see my comment in https://github.com/codeready-toolchain/toolchain-operator/pull/16#discussion_r350600268